stunnel/init.sls

```diff
@@ -16,29 +16,29 @@ stunnel_package:
     - user: {{ stunnel.user }}
     - makedirs: True
 
-{% if grains['os_family'] == 'FreeBSD' -%}
 {{ stunnel.conf_dir }}/conf.d:
   file.directory:
-    - user: root
-    - group: wheel
-
-{{ stunnel.conf_dir }}/conf.d/pid.conf:
-  file.absent: []
+    - user: {{ stunnel.user }}
+    - group: {{ stunnel.group }}
```

Comment:
It was in a FreeBSD-only section anyway, but I guess `init.sls` should be simplified anyway.